examples/next/docs/react/demo/src/renderers/ProgressBar.tsx

```diff
@@ -1,24 +1,33 @@
-import React from "react";
-import { BaseEditorComponent, HotEditorProps } from '@handsontable/react';
+import React from 'react';
+import { BaseEditorComponent } from '@handsontable/react';
 import { getRangeValue } from './utils';
+import { MESSAGE } from '../constants';
 
 const minAllowedValue = 0;
 const maxAllowedValue = 100;
 
-
-interface IProgressBarRendererProps extends HotEditorProps {
-  value: number;
-}
-
 export class ProgressBarRenderer extends BaseEditorComponent {
-  props!: IProgressBarRendererProps;
+  value: number;
+  isValid: boolean;
+  
+  constructor(props: any) {
```

Comment:
Avoiding `any` if the information about the props is easily available would probably be a good idea.

The renderer component receives [the same set of arguments the function-based](https://handsontable.com/docs/react-data-grid/cell-renderer/#declare-a-custom-renderer-as-a-function) renderer does, but as props.